spotbugsTestCases/src/java/bugIdeas/Ideas_2009_10_07.java

```diff
@@ -17,7 +17,7 @@ static int f(boolean b1, boolean b2, Object x) {
         return -x.hashCode();
     }
 
-    static int f(boolean b1, boolean b2) {
+    static int f2(boolean b1, boolean b2) {
```

Comment:
Renamed this function, so it doesn't have the same name as the one in line 5, and the bug can be checked easily.